pkg/engine/internal/imageverifier.go

```diff
@@ -243,8 +243,24 @@ func (iv *ImageVerifier) Verify(
 			iv.ivm.Add(image, true)
 			continue
 		}
+		found, err := iv.ivCache.Get(ctx, iv.policyContext.Policy(), iv.rule.Name, image)
```

Comment:
@eddycharly @realshuting @vishal-chdhry How can I initialize the cache to use in ImageVerifier  ??

Reply:
@hackeramitkumar what do you mean ?

Reply:
@eddycharly We have missed to add cache in mutateImageHandler  in [#7890](https://github.com/kyverno/kyverno/pull/7890). So imageverifier was not getting cache instance and causing nil pointer exception. I have fixed that. Now cache is working fine.